sdk/identity/Azure.Identity/src/DefaultAzureCredential.cs

```diff
@@ -38,12 +38,17 @@ public class DefaultAzureCredential : TokenCredential
         private TokenCredential[] _sources;
         private TokenCredential _credential;
 
+        /// <summary>
+        /// Creates a new instance of the <see cref="DefaultAzureCredential"/>.
+        /// </summary>
+        public DefaultAzureCredential() : this(false) { }
+
         /// <summary>
         /// Creates an instance of the DefaultAzureCredential class.
         /// </summary>
         /// <param name="includeInteractiveCredentials">Specifies whether credentials requiring user interaction will be included in the default authentication flow.</param>
-        public DefaultAzureCredential(bool includeInteractiveCredentials = false)
-            : this((includeInteractiveCredentials) ? new DefaultAzureCredentialOptions { ExcludeInteractiveBrowserCredential = !includeInteractiveCredentials } : null)
+        public DefaultAzureCredential(bool includeInteractiveCredentials)
+            : this(includeInteractiveCredentials ? new DefaultAzureCredentialOptions { ExcludeInteractiveBrowserCredential = false } : null)
```

Comment:
`ExcludeInteractiveBrowserCredentials` should still be set to `!includeInteractiveCredentials` rather than always setting it to `false`

Reply:
`includeInteractiveCredentials` is checked before assignment, so `ExcludeInteractiveBrowserCredential` can be only `false